### PR TITLE
Changes job ages

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
Changes job ages; lowers where applicable, getting rid of weird restrictions. 

Mostly for flavor. Does not lower job ages past where you would not conceivably be able to reach the minimum rank. (i.e an 18 y/o ensign)

### CODE REVIEW

File: maps/torch/job/corporate_jobs.dm [hunk 1]
```diff
@@ -48,7 +48,7 @@
 	selection_color = "#3d3d7f"
 	economic_power = 12
 	minimal_player_age = 2
-	minimum_character_age = list(SPECIES_HUMAN = 19)
+	minimum_character_age = list(SPECIES_HUMAN = 18)
```
Comment (Roland410) [inline maps/torch/job/corporate_jobs.dm:51]: This feels pretty borderline unless we assume that people are born to be bodyguards. 22-26 feels more appropriate.

File: maps/torch/job/medical_jobs.dm [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	department = "Medical"
 	department_flag = MED
 	minimal_player_age = 0
-	minimum_character_age = list(SPECIES_HUMAN = 29)
+	minimum_character_age = list(SPECIES_HUMAN = 25) //lowered to 25 because of Resident alt-title. if resident title is removed or someone changes age to somehow link to tile, raise back to 28
```
Comment (Roland410) [inline maps/torch/job/medical_jobs.dm:6]: Can med students even be residents at 25?

File: maps/torch/job/medical_jobs.dm [hunk 3]
```diff
@@ -50,7 +50,7 @@
 	spawn_positions = 3
 	supervisors = "Physicians and the Chief Medical Officer"
 	economic_power = 7
-	minimum_character_age = list(SPECIES_HUMAN = 19)
+	minimum_character_age = list(SPECIES_HUMAN = 18)
```
Comment (Roland410) [inline maps/torch/job/medical_jobs.dm:53]: This feels pretty young to me.

File: maps/torch/job/medical_jobs.dm [hunk 4]
```diff
@@ -141,7 +141,7 @@
 	supervisors = "the Chief Medical Officer, the Corporate Liaison and Medical Personnel"
 	selection_color = "#013d3b"
 	economic_power = 4
-	minimum_character_age = list(SPECIES_HUMAN = 25)
+	minimum_character_age = list(SPECIES_HUMAN = 21)
```
Comment (Roland410) [inline maps/torch/job/medical_jobs.dm:144]: Same as above.

File: maps/torch/job/medical_jobs.dm (hunk 2 above)
Comment (bearrrrrrrr) [reply maps/torch/job/medical_jobs.dm:6]: > can med students even be residents at 25

yes

File: maps/torch/job/corporate_jobs.dm (hunk 1 above)
Comment (bearrrrrrrr) [reply maps/torch/job/corporate_jobs.dm:51]: it was already 19 years old, i just removed the restriction /shrug

having it be 22+ will be fine and it's up to ppl

File: maps/torch/job/medical_jobs.dm (hunk 3 above)
Comment (bearrrrrrrr) [reply maps/torch/job/medical_jobs.dm:53]: ...there are 18 y/o emt's.

File: maps/torch/job/medical_jobs.dm (hunk 4 above)
Comment (bearrrrrrrr) [reply maps/torch/job/medical_jobs.dm:144]: eh it's literally just an arbitrary limit

also see above

**Comment by bearrrrrrrr** [comment]
you can literally be a 21 y/o FT. that is the min age.

i have adjusted the ages so that you will be the appropriate age at the min rank.

shrug

**Comment by JohnBigless** [comment]
> Combat tech's age is minimum 20 years
*"Would you be young and handsome all your life, Would you be young and handsome all your life?
Well the odds are pretty good that you'll be dead by twenty-two, so you'll be young and handsome all your life."*

**Comment by gilvian** [comment]
Christ some of those are bad, you should not be command age at this young, leave it as it is as a lot of people will be RPing some child prodigy command roles that were like "i graduated and now i have minimum of the required age so i am now the CMO on this ship"

**Comment by reklaw9** [comment]
there was one XO  that was -17-18Years old one round (he/she was a IPC,but i dont see that as a excuse)

**Comment by bearrrrrrrr** [comment]
@gilvian i'm not sure what to tell you

i will say this again; I have, _**again,**_ lowered the ages so that they are the minimum ages for their minimum rank.

Not sure what to tell you, ghost-rider.

**Comment by bearrrrrrrr** [comment]
If you're taking issue to the idea of some of these roles being played unrealistically by someone who has some weird snowflake, then please ahelp it. That's really all I can tell you.

**Comment by John-Dang** [comment]
Weezer here, lore developer of the server, saying at least from a lore-compliant angle the update is okay.

**Comment by John-Dang** [comment]
From a lore-compliant angle, this update is approved by the lore team's head developer, me.

**Comment by bearrrrrrrr** [comment]
@BurgerLUA it has lore feedback

it's g

also i don't know how to TRAVIS

**Pull request merged**